wordpress-mobile/WordPress-Android
# Fix Reader tab bar not retaining its scroll position

Fixes #11010

This should be considered a temporary fix. A larger rework is probably required to make the **Reader** take full advantage of the `TabLayout`. Please read the comments on the issue linked above for more details.

To test:

0. Make sure you're using the build variant thas has enabled IA flags.
1. Open the **Reader**.
2. Select an item off the right side of the tab bar. 
3. Notice that the tab becomes [centered](https://user-images.githubusercontent.com/47797566/71355040-18c6a500-257e-11ea-92af-82ad0ed19b0e.png).
4. Switch to the **Notifications** section of the app and then go back to the **Reader**.
5. Notice that the tab remains centered.

PR submission checklist:
